apps/site/package.json

```diff
@@ -19,7 +21,12 @@
     "storybook:build": "cross-env NODE_NO_WARNINGS=1 storybook build --quiet --webpack-stats-json",
     "test:unit": "cross-env NODE_NO_WARNINGS=1 jest",
     "test:unit:watch": "npm run test:unit -- --watch",
-    "test": "turbo test:unit"
+    "test": "turbo test:unit",
+    "cf": "npm run prebuild && npm run cf:build && npm run cf:preview",
+    "precf:build": "npm run prebuild",
+    "cf:build": "cross-env opennextjs-cloudflare && ln -s ../../pages ./.open-next/assets/",
```

Comment:
I'd rather avoid adding platform/vendor-specific commands to the package.json. Feel free to document these commands in a file. There are many new commands, and I do not want to complicate the DevEx of the average contributor/user.

Reply:
yeah this is quite work-in-progress, an ideal implementation, would override the `serve`, `start`, `deploy`, etc... scripts with Cloudflare ones, at this very experimental/hacks stage maybe it does make sense to keep the two distinct? 🤔  I don't have a huge preference on this actually 🤔